### ISSUE TITLE
inverse_square_root_schedule without warmup

### ISSUE THREAD

**Issue by HMJiangGatech** [pytorch/fairseq#1341]
If simply set warmup_updates=0, it will result in an error in the following code
https://github.com/pytorch/fairseq/blob/828c1ca7522278aa6c1eaf91fe0425b8f40dd832/fairseq/optim/lr_scheduler/inverse_square_root_schedule.py#L41

And there is no way to use inverse_square_root_schedule without warmup

**Comment by kokeman** [comment]
Does it work if "--warmup-updates" is  1 and "--lr" and "--warmup-init-lr" are the same value?
